src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -107,12 +107,12 @@
     ]},
 
     {node_metrics, [
-        {2, file_descriptors_open_limit, gauge, "Open file descriptors limit", fd_total},
-        {2, tcp_sockets_open_limit, gauge, "Open TCP sockets limit", sockets_total},
+        {2, erlang_process_max_fds, gauge, "Open file descriptors limit", fd_total},
+        {2, erlang_process_max_sockets, gauge, "Open TCP sockets limit", sockets_total},
         {2, memory_used_limit_bytes, gauge, "Memory high watermark in bytes", mem_limit},
```

Comment:
You're missing a prefixes here to indicate what's reporting this

Reply:
`rabbitmq_` gets appended to all metrics coming from this collector: https://github.com/rabbitmq/rabbitmq-prometheus/blob/17c685b73e974fcd251775af4eaf16a65ba7deaf/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl#L33

Reply:
In that case you shouldn't have the erlang_ as these are rabbitmq rather than erlang metrics.

Reply:
RabbitMQ is one of the many applications that runs within the Erlang VM, a single system process, typically `beam.smp`.

These are Erlang system process metrics, as observed (and cached for `n` seconds) by RabbitMQ.

Reply:
Your last comment confused me. Which of the following should this exporter expose?

1. `process_open_fds`
1. `rabbitmq_process_open_fds`
1. `rabbitmq_erlang_process_open_fds`

Reply:
process_open_fds if it's the number of fds the process is using.

Reply:
All metrics are cached and updated on an interval, typically every 5s. This is not something that we can change right now, it's a good future optimisation, for sure.

What should we use instead of `process_open_fds` if this metric is cached?

Reply:
It's rabbit specific, so rabbitmq_process_open_fds and make a note in the HELP that it's cached.